.gitignore

```diff
@@ -1,3 +1,4 @@
 /.idea/
 /target/
 /custom-output/
++/test-output/
```

Comment:
Please remove the "+" from the beginning. Thanks!

Reply:
aha, okay, ty!